ddr/lib/ddr-scanner/dwarf/DwarfScanner.cpp

```diff
@@ -64,6 +64,8 @@ class DwarfVisitor : public TypeVisitor
 	virtual DDR_RC visitUnion(UnionUDT *type) const;
 };
 
+const char * DwarfScanner::scanFileName = NULL;
```

Comment:
Why is this not initialized to NULL in the constructor?

Reply:
It's a `static` member.